projects/dsp-ui/src/lib/action/components/select-project/select-project.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="dsp-select-project-component">
+    <form class="form-content">
+        <mat-form-field class="large-field select-project">
+
+            <input
+                matInput
+                placeholder="Select project"
+                [matAutocomplete]="project"
+                [formControl]="selectProjectForm.controls['projectName']">
```

Comment:
I think this is the problem: 
> ERROR: projects/dsp-ui/src/lib/action/components/select-project/select-project.component.html:9:17 - error >TS2739: Type 'AbstractControl' is missing the following properties from type 'FormControl': registerOnChange, >registerOnDisabledChange, _applyFormState
 >          [formControl]="selectProjectForm.controls['projectName']">

I prefer this way:

```html
<form class="form-content" [formGroup]="selectProjectForm">
    <input
   [formControlName]="'projectName'">

</form>
```

Reply:
You can reproduce this locally when building in dev mode: `npm run build-lib-dev`.

This command builds the lib in dev mode (without the `--prod` flag) and uses

https://github.com/dasch-swiss/dsp-ui-lib/blob/fbb655e3441502edd9270d5437fa0253449250e0/projects/dsp-ui/tsconfig.lib.json#L20

see https://indepth.dev/a-look-at-major-features-in-the-angular-ivy-version-9-release#strict-template-type-checking

The build failed because the compiler could not determine whether you passed the correct type to `[FormVontrol]`.